package.json

```diff
@@ -21,9 +21,14 @@
     "query-string": "^6.4.2"
   },
   "devDependencies": {
+    "@edx/eslint-config": "^2.0.0",
     "coveralls": "^3.0.0",
-    "eslint": "^5.2.0",
-    "eslint-config-edx": "^4.0.4",
+    "eslint": "^8.10.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.29.2",
+    "eslint-plugin-react-hooks": "^4.3.0",
```

Comment:
Why do we need to add all these eslint dependencies instead of just upgrading eslint version?

Reply:
These packages were installed due to @edx/eslint-config. eslint-config-edx is deprecated and doesn't support node16 so we have to replace that with newer package @edx/eslint-config